pallet/account-migration/src/lib.rs

```diff
@@ -284,25 +283,37 @@ pub mod pallet {
 }
 pub use pallet::*;
 
+const SIGN_TIPS: &[u8] = b"\
+	Hi there from `account-migration`! \
+	Sign this message to prove you have access to this wallet and we'll do the migration for you. \
+	This won't cost you any. \
+	To stop hackers using your wallet, here's a unique message ID: \
+";
+
 fn sr25519_signable_message(
 	chain_id: u64,
 	spec_name: &[u8],
 	account_id_20: &AccountId20,
-) -> Message {
-	hashing::blake2_256(
-		&[
-			&hashing::blake2_256(
-				&[&chain_id.to_le_bytes(), spec_name, b"::account-migration"].concat(),
-			),
-			account_id_20.0.as_slice(),
-		]
-		.concat(),
-	)
+) -> Vec<u8> {
+	[
+		SIGN_TIPS,
+		hashing::blake2_256(
+			&[
+				&hashing::blake2_256(
+					&[&chain_id.to_le_bytes(), spec_name, b"::account-migration"].concat(),
+				),
+				account_id_20.0.as_slice(),
```

Comment:
But I would request to make account id also readable in the message, to avoid malicious app steal by replacing address.

No need to hash, I think.